onTruckPlugin/src/main/java/com/github/ontruck/Instruction.java

```diff
@@ -0,0 +1,52 @@
+package com.github.ontruck;
+
+import sun.reflect.generics.reflectiveObjects.NotImplementedException;
+
+class Instruction {
```

Comment:
Some passages in this class are quite unclear, I would like to see som comment explaining whats going on.